plugins/offboard/offboard_impl.cpp

```diff
@@ -106,6 +106,33 @@ void OffboardImpl::receive_command_result(MAVLinkCommands::Result result,
     }
 }
 
+void OffboardImpl::set_position_ned(Offboard::PositionNEDYaw position_ned_yaw)
+{
+    _mutex.lock();
```

Comment:
We should be using lock guards instead of manual lock and unlock but this needs to be changed in the whole plugin anyway.